st2reactor/st2reactor/garbage_collector/base.py

```diff
@@ -216,7 +217,8 @@ def _purge_action_executions_output(self):
         timestamp_str = isotime.format(dt=timestamp)
         LOG.info('Deleting action executions output objects older than: %s' % (timestamp_str))
 
-        assert timestamp < utc_now
+        if timestamp < utc_now:
+            raise ValueError('Calculated timestamp violates the utc now.')
```

Comment:
Bad comparison here, it's doing the exact opposite of what the previous assertion was doing.

The error message also needs to be tweaked - see my previous comment.

One of these is likely causing the CircleCI failures.